packages/core/src/formats.ts

```diff
@@ -5,17 +5,39 @@ export function date(
   locales: Locales,
   format: Intl.DateTimeFormatOptions = {}
 ): (value: string | Date) => string {
-  const formatter = new Intl.DateTimeFormat(locales, format)
-  return (value) => {
-    if (isString(value)) value = new Date(value)
-    return formatter.format(value)
+  try {
+    const formatter = new Intl.DateTimeFormat(locales, format)
+    return (value) => {
+      try {
+        if (isString(value)) value = new Date(value)
+        return formatter.format(value)
```

Comment:
In the case of the date, the error is thrown by `format` method when a value is in the wrong type.
In the case of the number, the error is thrown when formater is created without sufficient params. I tried few cases but the wrong data type seemed to be handled well by the number formater.

Neverthanless I couldn't find a prove that it can not throw an exception so I wrapped both cases in try/catch.

Reply:
I give it another tried and I removed any try/catch I couldn't reproduce in order to not lower the test coverage